src/interfaces/assistants_web/src/components/ConversationList/ConversationCard.tsx

```diff
@@ -33,15 +35,17 @@ const useMenuItems = ({ conversationId, name }: { conversationId: string; name:
 
   const menuItems: KebabMenuItem[] = [
     {
-      label: 'Edit title',
-      iconName: 'edit',
+      label: 'Share Chat',
+      iconName: 'share',
       onClick: () => {
-        editConversationTitle({ id: conversationId, title: name });
+        // editConversationTitle({ id: conversationId, title: name });
+        alert('TODO: Share Chat');
```

Comment:
why is this commented out? we do have the feature implemented for sharing conversations already